lambda_functions/two_way_sms/requirements-lambda.txt

```diff
@@ -0,0 +1,3 @@
+boto3
+psycopg2-binary==2.9.5
```

Comment:
do we need this in requirements if psycopg is a lambda layer?

Reply:
I didn't want to separate out dependencies being deployed in different ways.  If there was a layer for requests, then I would do both psycopg and requests as layers instead of using the requirements file.  Since there isn't one, I stuck with the requirements file so that all dependencies are defined in one place.

Reply:
I created a [ticket](https://github.com/department-of-veterans-affairs/notification-api/issues/998) for creating that layer.  Once that layer is created, we can create another ticket to update lambdas to use layers instead of requirements.txt